Makefile

```diff
@@ -81,7 +81,7 @@ deploy: push preflight-cluster
 	$(MAKE) deploy-only
 .PHONY: deploy
 
-deploy-only: preflight-dev-kubeconfig $(tools/kubectl) $(OSS_HOME)/manifests/emissary/emissary-crds.yaml $(boguschart_dir)
+deploy-only: preflight-dev-kubeconfig $(tools/kubectl) build-output/yaml-$(patsubst v%,%,$(VERSION)) $(boguschart_dir)
```

Comment:
Is it time to do 

```
VERSION_NO_V = $(patsubst v%,%,$(VERSION))
```

and just make this easier to read everywhere??

Reply:
I'm not _opposed_ but I'm fine with `$(patsubst v%,%,$(VERSION))` everywhere.